docs/pages/includes/edition-prereqs-tabs.mdx

```diff
@@ -1,74 +1,10 @@
-<Tabs>
-<TabItem scope={["oss"]} label="Teleport Community Edition">
-
-- A running Teleport cluster. For details on how to set this up, see the
-  [Getting Started](../index.mdx) guide.
+- A running Teleport cluster. If you want to get started with Teleport, [sign
+  up](https://goteleport.com/signup) for a free trial or [set up a demo
+  environment](../index.mdx).
 
 - The `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
 
-  See [Installation](../installation.mdx) for details.
-
-To check version information, run the `tctl version` and `tsh version` commands.
-For example:
-
-```code
-$ tctl version
-# Teleport v(=teleport.version=) git:(=teleport.git=) go(=teleport.golang=)
-  
-$ tsh version
-# Teleport v(=teleport.version=) go(=teleport.golang=)
-Proxy version: (=teleport.version=)
-Proxy: (=teleport.url=)
-```
-
-</TabItem>  
-<TabItem
-  scope={["enterprise"]} label="Teleport Enterprise">
-
-- A running Teleport Enterprise cluster. For details on how to set this up, see the Enterprise
-  [Getting Started](../choose-an-edition/teleport-enterprise/introduction.mdx) guide.
-
-- The Enterprise `tctl` admin tool and `tsh` client tool version >=
-  (=teleport.version=).
-  
-  You can download these tools by visiting your [Teleport
-  account workspace](https://teleport.sh).
-
-To check version information, run the `tctl version` and `tsh version` commands.
-For example:
-
-```code
-$ tctl version
-# Teleport Enterprise v(=teleport.version=) git:(=teleport.git=) go(=teleport.golang=)
-  
-$ tsh version
-# Teleport v(=teleport.version=) go(=teleport.golang=)
-Proxy version: (=teleport.version=)
-Proxy: (=teleport.url=)
-```
-
-</TabItem>
-<TabItem scope={["cloud"]}
-  label="Teleport Enterprise Cloud">
-
-- A Teleport Enterprise Cloud account. If you don't have an account, sign up to
-  begin a [free trial](https://goteleport.com/signup/). 
-
-- The Enterprise `tctl` admin tool and `tsh` client tool version >= (=cloud.version=).
-
-  You can download these tools from the [Cloud Downloads page](../choose-an-edition/teleport-cloud/downloads.mdx).
-
-To check version information, run the `tctl version` and `tsh version` commands.
-For example:
-
-```code
-$ tctl version
-# Teleport Enterprise v(=cloud.version=) git:(=teleport.git=) go(=teleport.golang=)
-  
-$ tsh version
-# Teleport v(=cloud.version=) go(=teleport.golang=)
-Proxy version: (=cloud.version=)
-Proxy: (=teleport.url=)
-```
-</TabItem>
-</Tabs>
+  On Teleport Enterprise, you must use the Enterprise version of `tctl`, which
+  you can download from your [Teleport account workspace](https://teleport.sh).
```

Comment:
We actually just removed enterprise `tctl` (moved all features to OSS), so this won't be necessary in 16+ https://github.com/gravitational/teleport.e/pull/3808

Reply:
Fixed in eac6a9c60b364839b2d855f6e5cacae6fb67eeb4